iina/PrefKeyBindingViewController.swift

```diff
@@ -264,17 +226,8 @@ class PrefKeyBindingViewController: NSViewController, PreferenceWindowEmbeddable
     } catch {
       Utility.showAlert("error_deleting_file", sheetWindow: view.window)
     }
-    userConfigs.removeValue(forKey: currentConfName)
-    Preference.set(userConfigs, for: Preference.Key.inputConfigs)
-    // load
-    if let index = userConfigNames.firstIndex(of: currentConfName) {
-      userConfigNames.remove(at: index)
-    }
-    confTableView.reloadData()
-    currentConfName = userConfigNames[0]
-    currentConfFilePath = getFilePath(forConfig: currentConfName)
-    confTableSelectRow(withTitle: currentConfName)
-    loadConfigFile()
+    // Fallback to default
+    loadConfigFile(configNames[0])
```

Comment:
I'm not confident using `[0]` here. What if someone changed the behavior of `configNames` later? it's better to do a `filter`, and if you don't like hard-coded config names, you can make it a constant.